pyqtgraph/imageview/ImageView.py

```diff
@@ -717,4 +717,14 @@ def menuClicked(self):
         if self.menu is None:
             self.buildMenu()
         self.menu.popup(QtGui.QCursor.pos())
-        
+
+    def setColorMap(self, colormap):
+        """Set the color map. *colormap* is an instance of ColorMap()"""
```

Comment:
Can you write this docstring following the convention outlined [here] (https://github.com/pyqtgraph/pyqtgraph/blob/develop/CONTRIBUTING.txt#L42)?